content/millennium/dstu2/care-provision/care-plan.md

```diff
@@ -32,20 +34,28 @@ _Implementation Notes_
  `patient`    | This, or one of `_id` or `subject`     | [`reference`] | Who care plan is for. Example: `patient=1316024`
  `subject`    | This, or one of `_id` or `patient`     | [`reference`] | Who care plan is for. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/1316024` or `subject:Patient=1316024`
  [`_count`]   | N                                      | [`number`]    | Number of results per page. Defaults to `10`. Capped at `10`.
+ `category`   | N                                      | [`token`]     | The scope of care plan being searched for. Examples: `category=assess-plan`, `category=http://argonaut.hl7.org|careteam`, `category=http://hl7.org/fhir/care-team-category|longitudinal`, or `category=careteam`
+ `context`    | N                                      | [`reference`] | The encounter id of the care plan. Example: `context=1138`
+
 
 Notes:
 
-- The `_id` parameter may not be provided at the same time as the `date`, `patient`, or `subject` parameters. When `_id` is provided, `_count` is ignored.
+- The `_id` parameter may not be provided at the same time as the `date`, `patient`, or `subject` parameters. When `_id` is provided, `_count` is ignored. When providing a category, `_id` will only allow a code of `assess-plan`.
 
 - The `patient` parameter is preferred when set simultaneously with `subject`.
 
 - The `date` parameter may be provided up to two times, and must use the `ge` or `le` prefixes. When provided twice, the lower value must have the `ge` prefix and the higher value must have the `le` prefix.
 
+- The `category` parameter only supports the codes `careteam`, `assess-plan`, `longitudinal`, and `encounter`. The `careteam` and `assess-plan` codes belong to the [Argonaut system]. `longitudinal` and `encounter` are defined by the [FHIR system].
+
 ### Response
 
 <%= headers 200 %>
 <%= json(:dstu2_care_plan_bundle) %>
 
+<%= headers 200, {Functionality: 'care plan with participants'} %>
```

Comment:
Do we want the description here to be something like 'CarePlan with participants (CareTeam)' ?